doc/release-notes.rst

```diff
@@ -265,6 +320,26 @@ Major Changes from Kraken
     - ``ceph tell <daemon> help`` will now return a usage summary.
     - ``ceph fs authorize`` creates a new client key with caps automatically
       set to access the given CephFS file system.
+    - The ``ceph health`` structured output (JSON or XML) no longer contains
+      'timechecks' section describing the time sync status.  This
+      information is now available via the 'ceph time-sync-status'
+      command.
+    - Certain extra fields in the ``ceph health`` structured output that
+      used to appear if the mons were low on disk space (which duplicated
+      the information in the normal health warning messages) are now gone.
+    - The ``ceph -w`` output no longer contains audit log entries by default.
+      Add a ``--watch-channel=audit`` or ``--watch-channel=*`` to see them.
+    - New "ceph -w" behavior - the "ceph -w" output no longer contains
+      I/O rates, available space, pg info, etc. because these are no
+      longer logged to the central log (which is what ``ceph -w``
+      shows). The same information can be obtained by running ``ceph pg
+      stat``; alternatively, I/O rates per pool can be determined using
+      ``ceph osd pool stats``. Although these commands do not
+      self-update like ``ceph -w`` did, they do have the ability to
+      return formatted output by providing a ``--format=<format>``
+      option.
```

Comment:
So, I still don't quite know what is going on, but the `ceph -w` behavior in luminous is definitely different and, I think, deserves a release note. The current one isn't quite right, though, because - AFAICT - in luminous it is still possible to get real-time I/O information from `ceph -w`. So, let's change it to this:

```
+    - New "ceph -w" behavior - the "ceph -w" output no longer updates
+      available space, pg info, etc. in real time because these are no
+      longer logged to the central log (which is what ``ceph -w``
+      shows). Some of the information can be obtained by running ``ceph pg
+      stat``. Also, if the I/O information provided by `ceph -w` is not fine-grained
+      enough, I/O rates per pool can be determined using
+      ``ceph osd pool stats``.
```

@jcsp What do you think? The current `ceph -w` behavior is a little confusing - it sits there, apparently listening to the central log, but in a quiescent cluster nothing appears. So far I tried creating and deleting rbd images, which I would think would create some I/O, but still nothing appeared. Still, I have anecdotal evidence that `ceph -w` does produce self-updating I/O information during performance tests.